Nos/Views/RepliesView.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  ThreadView.swift
+//  Nos
+//
+//  Created by Matthew Lorentz on 2/14/23.
+//
+
+import SwiftUI
+import SwiftUINavigation
+
+struct RepliesView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+    
+    @EnvironmentObject private var relayService: RelayService
+    @EnvironmentObject var router: Router
+
+    @State private var reply = ""
+    
+    @State private var alert: AlertState<Never>?
+    
+    var repliesRequest: FetchRequest<Event>
+    var replies: FetchedResults<Event> { repliesRequest.wrappedValue }
+    
+    var directReplies: [Event] {
+        replies.filter { ($0.eventReferences?.lastObject as? EventReference)?.eventId == note.identifier }
+    }
+    
+    init(note: Event) {
+        self.note = note
+        
+        if let rootReference = (note.eventReferences?.array as? [EventReference])?
+            .first(where: { $0.marker == "root" }),
```

Comment:
Is this marker string documented in the NIPS somewhere? Or did you just find it on real events? It's the first time I've noticed it.

Reply:
Yeah, this is the preferred format for e tags in text events, and it's what most tags I'm seeing in the wild look like now - https://github.com/nostr-protocol/nips/blob/master/10.md